BadHabbit/BadHabbit/UserStories/SingUpScreen/SignUpViewController.swift

```diff
@@ -0,0 +1,212 @@
+//
+//  SignUpViewController.swift
+//  BadHabbit
+//
+//  Created by Viktoria Misiulia on 19/07/2023.
+//
+
+import UIKit
+import SnapKit
+
+class SignUpViewController: UIViewController {
+    private var backgroundImageView: UIImageView!
+    private var welcomeLabel: UILabel!
+    private var appNameLabel: UILabel!
+    private var logoImageView: UIImageView!
+    private var logoLabel: UILabel!
+    private var phraseLabel: UILabel!
+    private var startButton: UIButton!
+ 
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        setupUI()
+    }
+}
+
+extension SignUpViewController {
+    private func setupUI() {
+        view.backgroundColor = UIColor(hexNumber: 0xCFCFCF)
+        
+        configureBackgroundImageView()
+        configureWelcomLabel()
+        configureAppNameLabel()
+        configureLogoImageView()
+        configurePhraseLabel()
+        configureStartButton()
+        
+        addSubviews([backgroundImageView,
+                     welcomeLabel,
+                     appNameLabel,
+                     logoImageView,
+                     phraseLabel,
+                     startButton
+                    ],
+                    toSuperview: view)
+        
+        setupViewConstraints()
+    }
+    
+    private func addSubviews(_ views: [UIView], toSuperview superview: UIView) {
+        views.forEach { subview in
+            superview.addSubview(subview)
+        }
+    }
+    
+    private func configureBackgroundImageView() {
+        backgroundImageView = UIImageView(image: UIImage(named: "mountains"))
+    }
+    
+    private func configureWelcomLabel() {
+        let label = UILabel()
+        
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.alignment = .center
+        paragraphStyle.lineHeightMultiple = 1.09
+
+        let attributes: [NSAttributedString.Key: Any] = [
+             .kern: -0.41,
+             .font: UIFont(name: FontName.SFProDisplayMedium.rawValue, size: 16.0)!,
+             .foregroundColor: UIColor(hexNumber: 0x000000),
+             .paragraphStyle: paragraphStyle
+        ]
+        
+        label.attributedText = NSAttributedString(string: "ДОБРО ПОЖАЛОВАТЬ В", attributes: attributes)
+
+        welcomeLabel = label
+    }
+    
+    private func configureAppNameLabel() {
+        let label = UILabel()
+        
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.alignment = .center
+        paragraphStyle.lineHeightMultiple = 1.09
+
+        let stringAttributes: [NSAttributedString.Key: Any] = [
+             .kern: -0.41,
+             .font: UIFont(name: FontName.SFProDisplayBold.rawValue, size: 24.0)!,
+             .foregroundColor: UIColor.black,
+             .paragraphStyle: paragraphStyle
+        ]
+        
+        label.attributedText = NSAttributedString(string: "HABIT HERO", attributes: stringAttributes)
```

Comment:
Why using attributes instead of setting this values to UILabel?

Reply:
I used attributes because I had to change the text kern.
Kern removed (based on your recommendations in pull request #10)